core/src/main/java/org/quickperf/annotation/FunctionalIteration.java

```diff
@@ -16,6 +16,24 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * The <code>FunctionalIteration</code> annotation disables all QuickPerf annotations on every scope. This annotation is
+ * intended to be used during the first implementation of a feature without bothering on performance fine tuning. Hence
```

Comment:
What do you think to replace 
"This annotation is intended to be used during the first implementation of a feature without bothering on performance fine tuning. Hence the naming of this annotation was designed to focus on implementation rather than the perf issues raised by QuickPerf."
by
"
This annotation is intended to be used during the first implementation of a feature without bothering on performance fine tuning. Hence the naming of this annotation was designed to focus on implementation rather than the perf issues raised by QuickPerf.
" ?

Reply:
Hi Jean,
I don't see any difference between the two options ?

Reply:
I am sorry Fabrice for this mistake.
I will be glad to have your opinion about the following version:
"
This annotation shows we are implementing the functional, not performance-related behavior. A typical use case is the development of a new feature. First, this annotation allows us to focus on the implementation of functional behavior. Global annotations are then disabled. In a second step, we remove it to re-enable global annotations. After, we may also add other QuickPerf annotations on the test method.
".